Source/Shift/Classes/SplitTransition.swift

```diff
@@ -88,19 +102,132 @@ public class SplitTransition: NSObject {
     }()
 
     /**
+     * In an interactive transition, splitOffset
+     * determines the vertical distance of the initial
+     * split when the cell is first tapped
+     */
+    public var splitOffset: CGFloat = 0.0
+
+    /**
```

Comment:
This is excellent advice. Do you use a plugin that generates the comment blocks for you or are you crafting your comments by hand?